test/common/http/conn_manager_impl_test.cc

```diff
@@ -1194,6 +1194,27 @@ TEST_F(HttpConnectionManagerImplTest, RouteShouldUseNormalizedHost) {
   filter_callbacks_.connection_.raiseEvent(Network::ConnectionEvent::RemoteClose);
 }
 
+// Observe host header w/o trailing dot in host when trailing dot removal is configured
```

Comment:
nit: here and below, end comments with period as comments should use proper grammar (even if other tests don't do this, no reason to not do it for new code :) )

Reply:
agree, will fix this.